drivers/usb/host/dwc_otg/dwc_otg_os_dep.h

```diff
@@ -27,6 +27,7 @@
 #include <linux/workqueue.h>
 #include <linux/stat.h>
 #include <linux/pci.h>
+#include <linux/compiler.h>
```

Comment:
Is this necessary? Here?

Reply:
The READ_ONCE macro used in dwc_otc_hcd.c:629